extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/HudiCatalogRangerSparkExtensionSuite.scala

```diff
@@ -326,6 +326,34 @@ class HudiCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
     }
   }
 
+  test("CompactionHoodiePathCommand / CompactionShowHoodiePathCommand") {
+    withSingleCallEnabled {
+      withCleanTmpResources(Seq.empty) {
+        val path1 = "hdfs://demo/test/hudi/path"
```

Comment:
can we also test a path suffixed with /

Reply:
also test null and ""

Reply:
> can we also test a path suffixed with /

Done

Reply:
> also test null and ""

Can't create such case from hudi sql command for such a case...